server/models/ride.ts

```diff
@@ -27,6 +27,11 @@ export type RideType = {
   endTime: string,
   rider: RiderType,
   driver?: DriverType,
+  recurring?: boolean,
+  recurringDays?: string[],
+  endDate?: string
+  deleted?: boolean,
+  edits?: string[],
```

Comment:
Could you also explain the reasoning behind wanting to save deleted rides instances to the `edits` array as well?

Reply:
The edits array has all the ride instances that are either being replaced or replacing an instance of the recurring ride. The normal ride instance that is being replaced (deleted = True) has to be stored in the edits so that it is not scheduled